splinterd/src/config/partial.rs

```diff
@@ -14,11 +14,22 @@
 
 use std::time::Duration;
 
+/// ConfigSource displays the source of configuration values, used to identify which of the various
+/// config modules were used to create a particular PartialConfig object.
+#[derive(Deserialize, Debug, Clone, PartialEq, Eq, Hash)]
+pub enum ConfigSource {
+    TomlFile,
```

Comment:
Consider calling this 'Toml' . Consider adding the toml filename so that can be output in the debug output table.

Reply:
Addressed.